frontend/src/index.css

```
@@ -168,6 +168,17 @@
   100% { opacity: 0.55; }
 }
 
+/* DA dot pulse when status changes via SSE */
+.animate-da-pulse {
+  animation: daPulse 1.5s ease-out;
+}
+
+@keyframes daPulse {
```

Stylelint (17.4.0)

[error] Expected keyframe name "daPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)